docs/source/conceptual-guides/endpoints.md

```diff
@@ -0,0 +1,180 @@
+# Understanding Lumigator Endpoints
+
+This document shows how current endpoints in lumigator work. If you want to customize lumigator with
```

Comment:
we should link to this from CONTRIBUTING

Reply:
Sure. This is just for understanding how endpoints work. I will add a new document describing how you can create a new one. Maybe we can link to the whole "Conceptual Guides" section, because later we're also have guides on how to add a new job, etc.